getting_started.md

```diff
@@ -0,0 +1,138 @@
+
+You will see upon adding a new translation the black cube gets shifted to the right immediately. This is because of the default vector specified in the translation. By default the field is disabled, but to edit these fields simply click on it. When finished editing, clicking elsewhere in the list will save the changes.
+
+![](resources/images/NeXusConstructor_008.png)
+
+The beam direction is conventionally along the z-axis, so use a vector of (0,0,1) and the set a distance of `-20m`. Distance units can be changed with the units field next to the value.
+
+![](resources/images/NeXusConstructor_009.png)
+
+
+### Adding a detector
+
+*Detectors* are a key component of any beamline. A detector comprised of an array of detecting pixels can be used to infer the angle that the neutrons/muons/photons scattered from the sample.
+
+Enter the "Add component" dialog and select "NXdetector" as the component type. Notice pixel data options appear.
+
+In this example our detector is comprised of a 16x16 grid of pixels.
+
+Select "Cylinder" as the "Shape type" and set both the height and radius to 1, and leave the units as "m".
+
+![](resources/images/AddComponent_010.png)
+
+We will select the "repeated single pixel shape" here and enter the rows and columns as 16. The row and column heights can be left as their default values.
+
+
+When hitting add, you will notice that the 3D view is updated should contain a grid of 16x16 cylinders. We now just need to move it backwards from the sample by applying another transformation of 5 metres along the z-axis.
+
+![](resources/images/NeXusConstructor_012.png)
+
+### Adding a Monitor
+
+We will add a monitor with a mesh shape to the instrument in this example. To do this, add another component.
+When the Add component Dialog is shown, select "NXmonitor" as the component type.
+
+After this, select "Mesh" as the "Shape Type".
+
+##### Loading shape from a CAD file
+Alternatively to cylinder shapes, mesh shapes can be used for components to describe their shape. Currently STL and OFF files are supported for mesh geometry.
+
+To use a mesh file for geometry, select the "Mesh" geometry type. This will show an option to browse for a file. As well as this, units can also be set to change the scale of the geometry.
+To test this, there are OFF and STL files in the `tests/` folder. Let's use the "cube.off" file.   
+
+![](resources/images/AddComponent_011.png)
+
+Again, this would need to be moved away from the sample if reflecting a real scenario. It may also need to be rotated. To add multiple transformations, select "Add rotation" or "Add translation" in the order you want them to be added.
+
+We now have a very basic example of a beamline instrument visualised in 3D as well as the resulting NeXus file. The NeXus Constructor can output to a JSON as well, which is used by the file-writer to aggregate event data and other live-streamed data to the file.
```

Comment:
Move the first sentence of this paragraph to the start of the next section and delete the rest.